group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupCoordinatorConfig.java

```diff
@@ -213,7 +215,8 @@ public class GroupCoordinatorConfig {
 
     public static final ConfigDef NEW_GROUP_CONFIG_DEF =  new ConfigDef()
             .define(GROUP_COORDINATOR_REBALANCE_PROTOCOLS_CONFIG, LIST, GROUP_COORDINATOR_REBALANCE_PROTOCOLS_DEFAULT,
-                    ConfigDef.ValidList.in(Utils.enumOptions(Group.GroupType.class)), MEDIUM, GROUP_COORDINATOR_REBALANCE_PROTOCOLS_DOC)
+                    ConfigDef.ValidList.in(Arrays.stream(Utils.enumOptions(Group.GroupType.class))
+                            .filter(groupType -> !groupType.equals(Group.GroupType.UNKNOWN.toString())).toArray(String[]::new)), MEDIUM, GROUP_COORDINATOR_REBALANCE_PROTOCOLS_DOC)
```

Comment:
nit: I wonder whether we should have a small helper method in `GroupType` to get the list of valid values. What do you think?